application-analytics-test/application-analytics-test-pageobjects/src/main/java/com/xwiki/analytics/test/po/AdminViewPage.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.analytics.test.po;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.xwiki.administration.test.po.AdministrationPage;
+import org.xwiki.test.ui.XWikiWebDriver;
+import org.xwiki.test.ui.po.ViewPage;
+
+public class AdminViewPage extends ViewPage
+{
+    private static final String TRACKING_CODE_ID = "Analytics.Code.ConfigurationClass_0_trackingCode";
+
+    private static final String AUTH_TOKEN_ID = "Analytics.Code.ConfigurationClass_0_authToken";
+
+    private static final String IS_SITE = "Analytics.Code.ConfigurationClass_0_siteId";
+
+    private static final String REQUEST_ADDRESS_ID = "Analytics.Code.ConfigurationClass_0_requestAddress";
+
+    private final XWikiWebDriver driver;
+
+    public AdminViewPage()
+    {
+        driver = getUtil().getDriver();
+    }
+
+    public AdminViewPage gotoAdminPage()
+    {
+        AdministrationPage administrationPage = AdministrationPage.gotoPage();
+        administrationPage.clickSection("Other", "Analytics");
+        return new AdminViewPage();
+    }
+
+    public AdminViewPage setTrackingCode(String value)
+    {
+        WebElement element = driver.findElement(By.id(TRACKING_CODE_ID));
```

Comment:
Check existing page objects. You can use the`` @FindBy`` annotation on ``WebElement`` fields of a class to inject the value so that you don't need to look up the elements yourself, unless this element is displayed dynamically after the page loads.